### PR TITLE
Prepared health-check-global for inclusion in C.G.C. documentation

### PR DESCRIPTION
**Pull request by sahanacp-hub**
Intending to use this snippet on the following page:
https://cloud.google.com//load-balancing/docs/health-checks

```release-note:none
```

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
health-check-region/main.tf
--- old/health-check-region/main.tf
+++ new/health-check-region/main.tf
@@ -16,7 +16,7 @@
 
 # [START cloudloadbalancing_regional_health_check]
 resource "google_compute_region_health_check" "default" {
-  name     = "tcp-health-check-region"
+  name = "tcp-health-check-region"
 
   timeout_sec        = 1
   check_interval_sec = 1
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
health-check-region/main.tf
--- old/health-check-region/main.tf
+++ new/health-check-region/main.tf
@@ -16,7 +16,7 @@
 
 # [START cloudloadbalancing_regional_health_check]
 resource "google_compute_region_health_check" "default" {
-  name     = "tcp-health-check-region"
+  name = "tcp-health-check-region"
 
   timeout_sec        = 1
   check_interval_sec = 1
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
health-check-region/main.tf
--- old/health-check-region/main.tf
+++ new/health-check-region/main.tf
@@ -16,7 +16,7 @@
 
 # [START cloudloadbalancing_regional_health_check]
 resource "google_compute_region_health_check" "default" {
-  name     = "tcp-health-check-region-west"
+  name = "tcp-health-check-region-west"
 
   timeout_sec        = 1
   check_interval_sec = 1
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
health-check-region/main.tf
--- old/health-check-region/main.tf
+++ new/health-check-region/main.tf
@@ -16,7 +16,7 @@
 
 # [START cloudloadbalancing_regional_health_check]
 resource "google_compute_region_health_check" "default" {
-  name     = "tcp-health-check-region-west"
+  name = "tcp-health-check-region-west"
 
   timeout_sec        = 1
   check_interval_sec = 1
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
health-check-region/main.tf
--- old/health-check-region/main.tf
+++ new/health-check-region/main.tf
@@ -16,7 +16,7 @@
 
 # [START cloudloadbalancing_regional_health_check]
 resource "google_compute_region_health_check" "tcp-health-check-region-west" {
-  name     = "tcp-health-check-region-west"
+  name = "tcp-health-check-region-west"
 
   timeout_sec        = 1
   check_interval_sec = 1
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by sahanacp-hub** [comment]
Please review the pull request. All 3 checks have passed.

File: health-check-region/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+resource "google_compute_region_health_check" "default" {
+  name               = "tcp-health-check-region-west"
+  timeout_sec        = 5
+  check_interval_sec = 5
+  tcp_health_check {
+    port = "80"
+  }
+  region = "us-west1"
+}
+# [END cloudloadbalancing_regional_health_check]# [END cloudloadbalancing_regional_health_check]
```
Comment (betsy-lichtenberg) [inline health-check-region/main.tf:27]: Duplicate end tag.

Comment (sahanacp-hub) [reply health-check-region/main.tf]: Removed the duplicate end tag.

**Comment by comment-bot-dev** [comment]
@sahanacp-hub
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by betsy-lichtenberg — APPROVED** [review]

**Pull request merged**